lib/helpers.js

```diff
@@ -18,3 +20,13 @@ export function getEditorsMap(editorUI) {
   })
   return {editors, diff, activePaths}
 }
+
+export function getMessageRange(message) {
+  return message.range ? (
+    message.range.constructor.name === 'Array' ? Range.fromObject(message.range) : message.range
+  ) : null
+}
+
+export function getMessageClass(message) {
+  return message.class + ' ' + message.type.toLowerCase()
```

Comment:
Shouldn't this be `message.className` due to https://github.com/atom-community/linter/commit/87e0f4dc24d142935001f8f45256e84f04fefd3a?